src/blocks/apt.rs

```diff
@@ -192,3 +198,38 @@ fn get_update_count(updates: &str) -> usize {
 fn has_matching_update(updates: &str, regex: &Regex) -> bool {
     updates.lines().any(|line| regex.is_match(line))
 }
+
+async fn get_update_count_ignore_phased(config_path: &str, updates: &str) -> Result<usize> {
+    let mut cnt = 0;
+
+    for update_line in updates.lines().filter(|line| line.contains("[upgradable")) {
+        if !is_phased_update(config_path, update_line).await? {
```

Comment:
You can also merge `get_update_count` and `get_update_count_ignore_phased` by doing something like

```rust
if !ignore_phased_updates || !is_phased_update(config_path, update_line).await? {
    cnt += 1;
}
```

Reply:
Makes sense. I don't know why I expected weirdness with an async term in an or-chain, but looks like it does get skipped you'd expect :+1: